.github/workflows/trigger-redeploy.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: CI
+
+on:
+  repository_dispatch:
+    types: [trigger-redeploy]
```

Comment:
I cant find whats triggering this event, should this be build? https://github.com/devfile/registry-support/pull/80/files#diff-003cb479ec4293d6e87970d83b072e03fbaf10e40f47378bc302a54a1ad18ed6R70

Reply:
So registry-support would trigger two workflows? This and the one in https://github.com/devfile/registry/blob/main/.github/workflows/pushimge-next.yaml?

Reply:
Oops, that should be a `build`

Reply:
Fixed.

Reply:
So would you need this workflow to wait for the image push workflow https://github.com/devfile/registry/blob/main/.github/workflows/pushimge-next.yaml to push the latest image, before triggering a deploy to staging? 

What if the trigger to redeploy staging uses the old image, since registry-support is going to dispatch the event and both these workflows will run in parallel

Reply:
Good question: Nope. AppSRE builds their own container image that’s used in the deployment. (quay.io/app-sre/devfile-index)

The registry deployment only runs after AppSRE’s Jenkins pipeline has successfully built and pushed the image.

We actually don’t need the image that’s built in the workflow you linked (and I’ve actually debated removing it)